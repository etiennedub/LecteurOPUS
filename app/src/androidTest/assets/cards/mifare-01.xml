<?xml version='1.0' encoding='UTF-8' standalone='yes' ?><Mifare><Pages><List index="0">04309C20</List><List index="1">2A334980</List><List index="2">D048E000</List><List index="3">000000FF</List><List index="4">00000000</List><List index="5">30DF4523</List><List index="6">00000000</List><List index="7">D0127FD8</List><List index="8">09C1382C</List><List index="9">82C10307</List><List index="10">91000000</List><List index="11">20287CBB</List><List index="12">01C1382C</List><List index="13">82C10307</List><List index="14">91000000</List><List index="15">202826BB</List></Pages></Mifare>