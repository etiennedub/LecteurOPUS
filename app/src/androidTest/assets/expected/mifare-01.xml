<?xml version='1.0' encoding='UTF-8' standalone='yes' ?><Card><Type>OCCASIONNELLE</Type><ID>0</ID><Trip><List index="0" Date="05/09/2016" Time="17:26" Transfert="false" OperatorId="20" BusId="16" OperatorName="RTC" BusName="LEBUS 28" /><List index="1" Date="05/09/2016" Time="17:26" Transfert="false" OperatorId="20" BusId="16" OperatorName="RTC" BusName="LEBUS 28" /></Trip><Contract><List index="0" Subscription="false" Valid="true" NbTicket="0" OperatorId="20" OperatorName="RTC" /></Contract></Card>